tests/backup/backup_locked_bucket_test.go

```diff
@@ -104,16 +105,15 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 			modes := [2]string{"GOVERNANCE", "COMPLIANCE"}
 			for _, provider := range providers {
 				for _, mode := range modes {
-					CredName := fmt.Sprintf("%s-%s-%v", "cred", provider, time.Now().Unix())
-					bucketName := fmt.Sprintf("%s-%s-%s-locked", provider, getGlobalLockedBucketName(provider), strings.ToLower(mode))
-					backupLocation = fmt.Sprintf("%s-%s-%s-lock", provider, getGlobalLockedBucketName(provider), strings.ToLower(mode))
+					bucketName := fmt.Sprintf("%s-%s-%v", getGlobalLockedBucketName(provider), strings.ToLower(mode), time.Now().Unix())
+					backupLocation = fmt.Sprintf("%s-%s-lock", getGlobalLockedBucketName(provider), strings.ToLower(mode))
```

Comment:
How about including time stamp?

Reply:
It'll have run hash passed to it through the env hence we won't need it.